src/udev/udev-builtin-kmod.c

```diff
@@ -56,31 +52,35 @@ static int builtin_kmod(UdevEvent *event, int argc, char *argv[]) {
 
 /* called at udev startup and reload */
 static int builtin_kmod_init(void) {
+        int r;
+
         if (ctx)
                 return 0;
 
-        ctx = kmod_new(NULL, NULL);
-        if (!ctx)
-                return -ENOMEM;
-
         log_debug("Loading kernel module index.");
-        kmod_set_log_fn(ctx, udev_kmod_log, NULL);
-        kmod_load_resources(ctx);
+        r = module_setup_context(&ctx);
+        if (r < 0)
+                return log_error_errno(r, "Failed to initialize libkmod context: %m");
```

Comment:
Hmm, other builtins don't seem to log above debug in `_init`?

Reply:
no quite true, the netif rename on calls naming_scheme() which logs at various higher levels.

Reply:
and the code around the udev_builtin_init() invocations tends to log loudly too.

i'd thus argue, we actually should fix the others to log above LOG_DEBUG too